aimmo-game/tests/functional/test_invulnerability_pickups_and_effects.py

```diff
@@ -47,4 +47,3 @@ def test_invulnerability_pickups_can_increase_resistance_to_2000(self):
 
         self.assertEqual(self.cell.avatar.resistance, 2000)
         self.assertEqual(self.cell.avatar, self.game.avatar_manager.get_avatar(1))
-        
```

Comment:
new line please

Reply:
!(https://i.imgur.com/311EKbV.png)

It's that thing again :-1:

Reply:
ok